src/utils/classifiers.ts

```diff
@@ -95,3 +95,15 @@ export const CREATE_TICKET_LINK =
   'https://sovryn.freshdesk.com/support/tickets/new';
 
 export const MILLION = 1000000;
+
+export const notificationServiceUrl = {
+  30: 'https://notify.sovryn.app/',
+  31: 'https://notify.test.sovryn.app/',
+};
+
+export const limitOrderUrl = {
+  30: 'https://orderbook.test.sovryn.app/limitOrder',
+  31: 'https://orderbook.sovryn.app/limitOrder',
+};
+
+export const tradeFormsGasLimit = 340000;
```

Comment:
This should be in `gasLimit` (in the same file).

Reply:
fixed